main.js

```diff
@@ -0,0 +1,270 @@
+const main = function() {
+  // body...
+
+  // This data will help with some logic in the game.
+  var data = {
+    board: [
+      ['1', '2','3'],
+      ['4', '5','6'],
+      ['7', '8','9']
+    ],
+    playerWins: false,
+    counter: 0,
+    positionsTaken: ['0']
+  }
+  // Get name of players. Save them in an array
+  const player1Name = prompt("What is player 1 name?");
```

Comment:
Though all of this is inside of the `main` closure, all of these are "globals" inside of your program. Could you have utilized your classes to do this?